open-source/overview.mdx

```diff
@@ -43,14 +43,9 @@ service:
       - "./wasm:/var/edgee/wasm"
 ```
 
-```nix nix flakes
-{
-  inputs.edgee.url = "github:edgee-cloud/edgee";
-}
-```
-
 ```shell cargo CLI
-cargo install edgee
+cargo build --release
+cargo run --release
```

Comment:
I have aligned this to what we have on https://github.com/edgee-cloud/edgee?tab=readme-ov-file#building-from-source

Reply:
In general, I think it would be a good idea to avoid duplicating this documentation at all, and just point people to the GitHub README - what do you think?

Reply:
I agree.